src/ui/flutter_app/android/app/build.gradle

```diff
@@ -64,22 +49,19 @@ android {
         }
     }
 
-    signingConfigs {
-        release {
-            keyAlias keystoreProperties['keyAlias']
-            keyPassword keystoreProperties['keyPassword']
-            storeFile keystoreProperties['storeFile'] ? file(keystoreProperties['storeFile']) : null
-            storePassword keystoreProperties['storePassword']
-        }
-    }
+
     buildTypes {
         debug {
             signingConfig signingConfigs.debug
-            //applicationIdSuffix ".debug"
-            //debuggable true
+            minifyEnabled true
+            shrinkResources true
+
         }
         release {
-            signingConfig signingConfigs.release
+            signingConfig signingConfigs.debug
```

Comment:
Why do you turn on the debug switch on the Release version of the APK instead? Does this degrade performance?